lessons/lesson-3/concepts.md

```diff
@@ -1,5 +1,19 @@
 # Concepts
 
+Hand interaction is a very natural way to interact with 3D models. As an interaction model, the use of hands is great for new users coupled with voice, eye tracking or head gaze. There is a low learning curve and consistent UX across hand tracking.
+
+Hand interaction may be suitable for your mixed reality application if any of the scenarios below apply to your mixed reality application:
```

Comment:
This could be it's own question, as well as line 11.

Reply:
I can pull out lines 5-9 into it's own question.

And I can pull out lines 11-15 into it's own question.